crates/adapters/src/transport/kafka/ft/output.rs

```diff
@@ -16,14 +16,13 @@ use rdkafka::{
     ClientConfig, ClientContext, Message,
 };
 use serde::{Deserialize, Serialize};
-use std::{
-    cmp::max,
-    sync::{Condvar, Mutex, RwLock},
-    time::Duration,
-};
+use std::{cmp::max, sync::RwLock, time::Duration};
 
 use super::{count_partitions_in_topic, CommonConfig, Ctp, DataConsumerContext};
 
+/// Send retry frequency.
+const OUTPUT_POLLING_INTERVAL: Duration = Duration::from_millis(10);
```

Comment:
shouldn't this be using some exponential back-off too?

Reply:
yeah, I'll do that

Reply:
isn't there some code to do that? you'd think this is a very common pattern

Reply:
there is a crate, but we don't need a full-blown randomized backoff, as we're not racing for access to the resource.